osm2pgsql.lua

```diff
@@ -499,128 +499,116 @@ function osm2pgsql.process_node(object)
-        output_hstore['place:AR'] = 'region'
-        output_hstore['place:BD'] = 'region'
-        output_hstore['place:BR'] = 'region'
-        output_hstore['place:CN'] = 'region'
-        output_hstore['place:DE'] = 'region'
-        output_hstore['place:EG'] = 'region'
-        output_hstore['place:GB'] = 'region'
-        output_hstore['place:GR'] = 'region'
-        output_hstore['place:ID'] = 'region'
-        output_hstore['place:IL'] = 'region'
-        output_hstore['place:IN'] = 'region'
-        output_hstore['place:IT'] = 'region'
-        output_hstore['place:JP'] = 'region'
-        output_hstore['place:KO'] = 'region'
-        output_hstore['place:MA'] = 'region'
-        output_hstore['place:NL'] = 'region'
-        output_hstore['place:NP'] = 'region'
-        output_hstore['place:PK'] = 'region'
-        output_hstore['place:PL'] = 'region'
-        output_hstore['place:PS'] = 'region'
-        output_hstore['place:PT'] = 'region'
-        output_hstore['place:SA'] = 'region'
-        output_hstore['place:SE'] = 'region'
-        output_hstore['place:TR'] = 'region'
-        output_hstore['place:TW'] = 'region'
-        output_hstore['place:UA'] = 'region'
-        output_hstore['place:VN'] = 'region'
+        output_hstore['place'] = nil
     end
--- Turn off South Ossetia label for most countries
+-- Turn off South Ossetia label
     if object.tags.place and object.tags.wikidata == 'Q23427' then
-        output_hstore['place:AR'] = 'region'
-        output_hstore['place:BD'] = 'region'
-        output_hstore['place:BR'] = 'region'
-        output_hstore['place:CN'] = 'region'
-        output_hstore['place:DE'] = 'region'
-        output_hstore['place:EG'] = 'region'
-        output_hstore['place:GB'] = 'region'
-        output_hstore['place:GR'] = 'region'
-        output_hstore['place:ID'] = 'region'
-        output_hstore['place:IL'] = 'region'
-        output_hstore['place:IN'] = 'region'
-        output_hstore['place:IT'] = 'region'
-        output_hstore['place:JP'] = 'region'
-        output_hstore['place:KO'] = 'region'
-        output_hstore['place:MA'] = 'region'
-        output_hstore['place:NL'] = 'region'
-        output_hstore['place:NP'] = 'region'
-        output_hstore['place:PK'] = 'region'
-        output_hstore['place:PL'] = 'region'
-        output_hstore['place:PS'] = 'region'
-        output_hstore['place:PT'] = 'region'
-        output_hstore['place:SA'] = 'region'
-        output_hstore['place:SE'] = 'region'
-        output_hstore['place:TR'] = 'region'
-        output_hstore['place:TW'] = 'region'
-        output_hstore['place:UA'] = 'region'
-        output_hstore['place:VN'] = 'region'
+        output_hstore['place'] = nil
     end
--- Turn off Nagorno-Karabakh label for most countries
+-- Turn off Nagorno-Karabakh label
     if object.tags.place and object.tags.wikidata == 'Q2397204' then
-        output_hstore['place:AR'] = 'region'
```

Comment:
This one is more like South Ossetia in that Nagorno-Karabakh isn't a region label for Azerbaijan... so I think you want to set the default place tag to `country` so it'll get exported as `unrecognized`...

But you'll want to add **place:RU** = `region` since Russia recognizes it as a ... something between country and region.

Remove the others.